R/modify_pd.R

```diff
@@ -0,0 +1,53 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler()] or
+#'   [create_filler_nested()].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  if (any(pd$token == "')'")) {
+    start <- 2
+    stop <- nrow(pd) - 1
+  } else {
+    start <- stop <- 0
+  }
+  pd$indent <- ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)
```

Comment:
Not sure what you @krlmlr mean. We need to assign a value to `pd$indent` anyways, since this column is used in the recursion of `serialize_parse_data_nested_helper`. Otherwise we have to change this function so it can handle the case for which the column does not exist, which I think is not a good idea.